changelog.html

```diff
@@ -32,5 +32,10 @@ <h4>17th of December, 2021 - 2021-12-17</h4>
 		<li>Added the first entry of the changelog</li>
 		<li>Updated libraries, bot ported to Discord.js V13</li>
 	      </ul>
+	  <br> <!-- Optional --> 
+	  <h4>21st of January, 2022 - 2022-01-21</h4>
+	      <ul>
+		<li>Made RoBATS an open source project and created the GitHub repository.</li>	
+	      </ul>
 	      
 </body>
```

Comment:
also you should add a </html> in the end

Reply:
If I didn't, it was because I was distracted, lmao